content/10.zk-stack/20.running-a-zk-chain/30.custom-base-tokens.md

```diff
@@ -0,0 +1,82 @@
+---
+title: Custom Base Tokens
+description:
+---
+
+You can customize your ZK chain by changing the base token used for gas from ETH to an ERC20 token.
+
+With the initial release of ZK stack, base tokens must be whitelisted for use as a base token for a chain.
+The whitelisted addresses are stored and checked in the `BridgeHub` contract on the L1.
```

Comment:
_I'm still new to ZK chains so my knowledge is almost non-existent, so my questions might be dumb questions._

By "addresses" this means that there can be more than one base token for a chain? Or that every chain's base token address is available on the L1?

Reply:
I might also be wrong here, but I think every chain's base token address is available on the L1 through this `BridgeHub` contract